cmd/clusterctl/test/run-e2e.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
General question: what about the creation of the local overrides? should it be part of this script?